payments/stripe/forms.py

```diff
@@ -44,7 +39,7 @@ def clean(self):
                             self.payment.billing_last_name,
                             self.payment.billing_first_name,
                         ),
-                        "metadata": self.payment.order.get_metadata(),
+                        "metadata": "Order #{}".format(self.payment.pk),
```

Comment:
This change will break existing integrations which rely on `.order.get_metadata()`. See https://github.com/jazzband/django-payments/issues/346#issuecomment-1515117386

Reply:
I found it here 
[e9f66dcc0814f2c9d445fa1011bba7abeae12c42](https://github.com/jazzband/django-payments/commit/e9f66dcc0814f2c9d445fa1011bba7abeae12c42#diff-f3dff9e6592453daa749a3035de2f9104067721cc260b2d3926835959671aaddR44)

But couldn't find the definition, i'll add a try/except to get function result, otherwise use pk or token